config/packages/reset_password.yaml

```diff
@@ -1,2 +1,2 @@
 symfonycasts_reset_password:
-  request_password_repository: App\Repository\ResetPasswordRequestRepository
\ No newline at end of file
+    request_password_repository: App\Repository\ResetPasswordRequestRepository
```

Comment:
Fixed in https://github.com/jrushlow/reset-password-bundle-flex/commit/0387dcb6259479c71841edbf9b37fde00b812b17